templates/cowcheck/0/rancher-compose.yml

```diff
@@ -0,0 +1,40 @@
+version: '2'
+catalog:
+  name: cowcheck
+  version: 0.2.0
+  description: A microservice for checking the health of a Rancher node
+  minimum_rancher_version: v.1.6.0
+  questions:
+    - variable: POLL_INTERVAL
+      label: POLL_INTERVAL
+      description: | Frequency in seconds to run checks
```

Comment:
Description fields are making build fails. If you use `|` you need new line.
Please, review all description fields.
```
-      description: | Frequency in seconds to run checks
+      description: | 
+        Frequency in seconds to run checks
OR
+      description: Frequency in seconds to run checks
```